api/image.go

```diff
@@ -386,58 +541,37 @@ func generateImagePublishEvents(image *models.Image) (events []*event.ImagePubli
 	return events, nil
 }
 
-//ImageUploadedEvent returns an ImageUploaded event for the provided image ID and upload path
-var ImageUploadedEvent = func(imageID, uploadPath string) *event.ImageUploaded {
-	return &event.ImageUploaded{
-		ImageID: imageID,
-		Path:    uploadPath,
+// unlockImage unlocks the provided image lockID and logs any error with WARN state
+func (api *API) unlockImage(ctx context.Context, lockID string) {
+	if err := api.mongoDB.UnlockImage(lockID); err != nil {
+		log.Event(ctx, "error unlocking mongoDB lock for an image resource", log.WARN, log.Data{"lockID": lockID})
 	}
 }
 
-// ImagePublishedEvent returns an ImagePublished event for the provided path
-var ImagePublishedEvent = func(path string) *event.ImagePublished {
-	return &event.ImagePublished{
-		SrcPath: path,
-		DstPath: path,
-	}
-}
+/// TODO Remove Deprecated handlers…
```

Comment:
NB. The handlers below are no longer part of the swagger specification. 
- `UploadImageHandler` will now become the `PUT /images/{id}` endpoint and will need a small amount of rework under a different task
- `ImportVariantHandler` and `CompleteVariantHandler` will be combined and form the `PUT /images/{id}/downloads/{variant}` endpoint.

The code has been left in place until this change but the handlers are not used by any route.